canonical/maas-ui
# feat(machines): display filter options from API

## Done

- Display filters in the machine list dropdown that have been fetched from the API.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the machine list.
- Open the "Filters:" dropdown (next to the search box).
- Click on one of the filter headers and the options should load.
- Clicking on one of the options should populate the search box (not all filters are currently working, but that'll be handled in https://github.com/canonical/app-tribe/issues/1263).

## Fixes

Fixes: canonical/app-tribe#1124.
